src/client/client.cpp

```diff
@@ -1891,6 +1891,18 @@ void Client::makeScreenshot()
 	raw_image->drop();
 }
 
+float Client::getReward() {
+	float reward = 0.0;
+	//ClientScripting *scr = getScript();
```

Comment:
This is commented because it is throwing LUA errors. @neverix I guess this is because I don't have the reward mod installed that you used?

Reply:
Probably, maybe the issue is in the config